yarn-project/archiver/src/archiver/archiver.test.ts

```diff
@@ -133,30 +120,11 @@ function makeL2BlockProcessedEvent(l1BlockNum: bigint, l2BlockNum: bigint) {
   } as Log<bigint, number, undefined, typeof RollupAbi, 'L2BlockProcessed'>;
 }
 
-/**
- * Makes a fake UnverifiedData event for testing purposes.
- * @param l1BlockNum - L1 block number.
- * @param l2Block - The l2Block this event is associated with.
- * @returns An UnverifiedData event log.
- */
-function makeUnverifiedDataEvent(l1BlockNum: bigint, l2Block: L2Block) {
-  return {
-    blockNumber: l1BlockNum,
-    args: {
-      l2BlockNum: BigInt(l2Block.number),
-      l2BlockHash: `0x${l2Block.getCalldataHash().toString('hex')}`,
-      sender: EthAddress.random().toString(),
-      data: '0x' + createRandomUnverifiedData(16).toString('hex'),
-    },
-    transactionHash: `0x${l2Block.number}`,
-  } as Log<bigint, number, undefined, typeof UnverifiedDataEmitterAbi, 'UnverifiedData'>;
-}
-
 /**
  * Makes a fake ContractDeployed event for testing purposes.
  * @param l1BlockNum - L1 block number.
  * @param l2Block - The l2Block this event is associated with.
- * @returns An UnverifiedData event log.
+ * @returns An EncryptedLogs event log.
  */
 function makeContractDeployedEvent(l1BlockNum: bigint, l2Block: L2Block) {
```

Comment:
```suggestion
function makeContractDeploymentEvent(l1BlockNum: bigint, l2Block: L2Block) {
```

I think it would be nice to keep the function name consistent with the event name.